frontend/styles/variables.scss

```diff
@@ -87,7 +87,8 @@ $spacers: (
 // Define the maximum width of `.container` for different screen sizes.
 $container-max-widths: (
   sm: 540px,
-  md: 780px, // changed from original Bootstrap 720px for extra space!
+  md: 780px,
+  // changed from original Bootstrap 720px for extra space!
```

Comment:
Shouldn't this go above the md-definition?

Reply:
Seems like my linter moved it down from the end of the line.